aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_distro.py

```diff
@@ -117,5 +115,7 @@ def _configure(self, **kwargs):
             # Disable AWS Application Signals by default
             os.environ.setdefault(APPLICATION_SIGNALS_ENABLED_CONFIG, "false")
 
+        super(AwsOpenTelemetryDistro, self)._configure()
```

Comment:
Curious on why did we have to move this down?

Reply:
> Curious on why did we have to move this down?

`super(AwsOpenTelemetryDistro, self)._configure()` will set all OTel Exporters to `otlp` by default only if they're not set. So we want to set it first (esp for OTel Metrics Exp) and then apply this defaulting logic